release-orb.sh

```diff
@@ -1,11 +1,22 @@
 #!/bin/bash
 
-echo "Starting release."
+function version_gt() {
+    test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1";
+}
 
+echo "Starting release."
 
 command -v circleci >/dev/null 2>&1 || { echo >&2 "I require circleci but it's not installed.  Aborting."; exit 1; }
 echo "Found circleci command."
 
+cli_version=$(circleci version | cut -d+ -f1)
+required_version=0.1.5705
```

Comment:
Do you want this to be a static version?
`circleci update check` could do some of the work

Reply:
I think so, yeah.  If we change the functionality of the script, we can change the minimum version.  That update check doesn't work for everyone:

```
└─ circleci update check
`update` is not available because this tool was installed using `homebrew`.
Please consult the package manager's documentation on how to update the CLI.
```

Reply:
Boo.. Carry on.